src/components/PeopleTable.tsx

```diff
@@ -10,635 +33,90 @@ export const PeopleTable = () => {
-            <a
-              className="has-text-danger"
-              href="#/people/maria-van-brussel-1801"
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
+        {people.map(
+          ({
+            name,
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+            slug: personSlug,
+          }) => (
+            <tr
+              data-cy="person"
+              key={personSlug}
+              className={classNames({
+                'has-background-warning': slug === personSlug,
+              })}
+            >
+              <td>
+                <Link
+                  to={`/people/${personSlug}`}
+                  className={classNames({
+                    'has-text-danger': sex === Sex.Female,
+                  })}
+                >
+                  {name}
+                </Link>
+              </td>
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+              <td>
+                {motherName && findPersonByName(motherName)?.slug ? (
+                  <Link
+                    to={`/people/${findPersonByName(motherName)?.slug}`}
+                    className="has-text-danger"
+                  >
+                    {motherName}
+                  </Link>
+                ) : (
+                  <span>{motherName || '-'}</span>
+                )}
+              </td>
+              <td>
+                {fatherName && findPersonByName(fatherName)?.slug ? (
+                  <Link to={`/people/${findPersonByName(fatherName)?.slug}`}>
```

Comment:
you can save the result of the `findPersonByName(fatherName)` function to some var and reuse it everywhere instead of calling it twice